### PR TITLE
Adding support for CreateFileShareTarget as part of CreateFileShare call

### PR DESCRIPTION
**Pull request by sameshai**

### CODE REVIEW

File: file/provider/create_volume.go [hunk 1]
```diff
@@ -63,6 +63,21 @@ func (vpcs *VPCSession) CreateVolume(volumeRequest provider.Volume) (volumeRespo
 		},
 	}
 
+	// Check for VPC ID, SubnetID or PrimaryIPID either of the one is mandatory for VolumeAccessPoint/FileShareTarget creation
```
Comment (skorati) [inline file/provider/create_volume.go:66]: Could you please add more documentation for this condition.

Comment (sameshai) [reply file/provider/create_volume.go:66]: Done

**Review by arahamad — APPROVED** [review]
lgtm, please verify these changes and share the result in the PR as well

File: file/provider/create_volume.go (hunk 1 above)
Comment (skorati) [reply file/provider/create_volume.go:66]: Than you @sameshai

**Review by skorati — APPROVED** [review]
looks good.

**Pull request merged**